scripts/helper_scripts/unipept-database-rs/src/taxons_uniprots_tables/utils.rs

```diff
@@ -0,0 +1,16 @@
+use anyhow::{Context, Result};
+use chrono::{DateTime, Utc};
+use std::time::{SystemTime, UNIX_EPOCH};
+
+pub fn now() -> Result<u128> {
+    Ok(SystemTime::now()
+        .duration_since(UNIX_EPOCH)
```

Comment:
There will only occur an error here if [`SystemTime#duration_since`](https://doc.rust-lang.org/std/time/struct.SystemTime.html#method.duration_since) is called with `SystemTime::now()` being before `UNIX_EPOCH`. Which I don't think is realistically possible.

I think there is no risk in making this an `expect()` to simplify these functions.

@ninewise what do you think?

Reply:
Agreed. For an internal tools it's OK to panick in very unlikely circumstancs.